src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,27 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.Table;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
+@Table (name = "movies")
 public class Movie implements Cloneable {
```

Comment:
Implementing Cloneable and providing a clone method is not a typical practice for entity classes in Hibernate. Hibernate manages entities and their lifecycle, and cloning objects manually can lead to unexpected behavior, especially with managed associations and proxies. Consider removing the Cloneable interface and the clone method unless there's a specific requirement for it.